pt-BR/mail/chrome/messenger/messengercompose/addressingWidgetOverlay.dtd

```diff
@@ -1,9 +1,8 @@
 <!-- address labels -->
 <!--LOCALIZATION NOTE addressingWidgetOverlay.dtd The basic mail/news composition headers as they are seen in UI -->
-<!ENTITY toAddr.label   "Para:"> 
-<!ENTITY ccAddr.label   "Cc:"> 
-<!ENTITY bccAddr.label  "Cco:"> 
-<!ENTITY replyAddr.label   "Responder a:"> 
-<!ENTITY newsgroupsAddr.label   "Newsgroup:"> 
-<!ENTITY followupAddr.label   "Followup-To:"> 
-
+<!ENTITY toAddr.label   "Para:">
+<!ENTITY ccAddr.label   "Cc:">
+<!ENTITY bccAddr.label  "Cco:">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY bccAddr.label  "Bcc:">
@brunotiussi please don't translate the string Bcc.